pypit/scripts/coadd_1dspec.py

```diff
@@ -41,7 +41,7 @@ def main(args, unit_test=False):
     files = []
     for ifl in filelist:
         if '*' in ifl:
-            files.append(glob.glob(ifl))
```

Comment:
I think this should be back to the way it was before. Imagine the case where filelist contains

```
another.fits
blah*.fits
```

The new code would ignore `another.fits`, whereas the old code would include another.fits as well as all of `blah*.fits`... I think...